packages/create-package/README.md

```diff
@@ -6,17 +6,24 @@
 
 Provides an `npm init` script for creating new "UI5 Web Components" packages.
 
-## Usage
+## Usage with npm
+
+```
+Usage:
+    npm init @ui5/webcomponents-package [OPTIONS]
+Options:
+    --name <string>     - defines package name
+    --port <number>     - defines default package server port
+    --tag <string>      - defines predefined custom element tag name
+    --enable-typescript - enables TypeScript support to the package
+```
 
-with `npm`
-
-- `npm init @ui5/webcomponents-package` (the user will be asked for **name**, **port**, **tag**);
-- `npm init @ui5/webcomponents-package <NEW-PACKAGE-NAME>` (the user will be asked for **port** and **tag** only).
+The script creates a new directory, and fills it with a `package.json` file and all necessary source files, and resources for a new
+components package.
 
-And with `yarn`
+## Usage with yarn

```

Comment:
we need to at least say that the options are available with yarn as well, if not repeating them as above